www_src/pages/project/project.jsx

```diff
@@ -241,11 +175,77 @@ var Project = React.createClass({
     });
   },
 
-  zoomOut: function () {
-    this.setState({zoom: this.state.zoom / 2});
+  touchState: {
+    didMove: false,
+    startX: 0,
+    startY: 0,
+    endX: 0,
+    endY: 0
+  },
+
+  onTouchStart: function (event) {
```

Comment:
This is just for the swiping stuff now, since all the other interactions are handled by panzoom